rfcs/rfc-standalone-annual-report.md

```diff
@@ -0,0 +1,81 @@
+
+
+# Summary
+
+Create a standalone Annual Report filing and Beneficial Ownership registration to prove out the workflow and UX for capturing the data required for the Transparency Registry.
+
+# Motivation
```

Comment:
It's legislated that they have to provide ben owner info in an annual report; it's not legislated they have to file at time of creation.  But that doesn't mean the annual report needs to move from the Filings UI.  What is the extra value of moving the AR from the Filings UI to the RTR repo?  Businesses will launch any other filings/company changes from their business dashboard, why are we breaking the pattern?  Could we keep it in the Filings UI or Edit UI and then link out to the RTR repo to complete Ben owner info and then come back to finish off AR?

AR output should not include any ben owner info.

Reply:
Having AR as something that is directly linkable and that we can make available today.
If we leave it in FilingsUI, then only business that have been migrated to Lear can use it.

I'm only suggesting AR, as it is a really small filing (the date) and we could send that info to Colin (already built into ColinAPI) or to Lear fairly easily.

This way we could launch or move into some UX Research now, without having to wait for the BCA types to migrate over.

Reply:
We can't use it until OIC passes is early 2025.  By that time all corporations will be on Lear.

Reply:
Based upon recent comments, I had thought we could start gathering in a non-required way in early 2024. Same as LOTR gathered data for 4 years before becoming mandatory and enforceable.

Reply:
I wasn't suggesting AR include and BenOwner info. As we don't want to combine any of that data into the business registry, it would also give us a nice way to test flows through the linked apps, much like we link between the homepage, dashboards and filings.